lib/services/cups.pm

```diff
@@ -81,6 +81,16 @@ sub check_function {
     record_info "access_log", "Access log should containt successful job submits";
     assert_script_run 'grep "Send-Document successful-ok" /var/log/cups/access_log';
 
+    # Check error log
+    my $error_log = "/var/log/cups/error_log";
+
+    if (script_run("test -f $error_log") == 0) {
```

Comment:
Wouldn't be -s better ? file is greater than zero ?

Reply:
The error_log is not created unless there are errors, otherwise it does not exist, so it will never be empty. However it makes more sense when one reads the code so I did change it, thanks for the suggestion!